src/wallet/wallet2.h

```diff
@@ -673,6 +679,26 @@ namespace tools
     uint64_t get_fee_multiplier(uint32_t priority, int fee_algorithm = -1);
     uint64_t get_per_kb_fee();
 
+    /*
+     * "attributes" are a mechanism to store an arbitrary number of string values
+     * on the level of the wallet as a whole, identified by keys. Their introduction,
+     * technically the unordered map m_attributes stored as part of a wallet file,
+     * led to a new wallet file version, but now new singular pieces of info may be added
+     * without the need for a new version.
+     *
+     * The first and so far only value stored as such an attribute is the description.
+     * It's stored under the internal standard key ATTRIBUTE_DESCRIPTION (see method set_description).
+     *
+     * The mechanism is now private to wallet2. For the future it would be feasible to open
+     * it to all clients however and allow them to use it for storing basically any single
```

Comment:
This is public now.

Reply:
Yes, of course. I adjusted the comments. It's already a little embarassing how many rounds I need to get this PR right ... hope that's it now :)